actions/run_action.go

```diff
@@ -68,6 +70,22 @@ func (run *RunAction) Verify(context *debos.DebosContext) error {
 	if run.Script == "" && run.Command == "" {
 		return errors.New("Script and Command both cannot be empty")
 	}
+
+	if run.Script != "" {
+		argv := strings.SplitN(run.Script, " ", 2)
+		script := debos.CleanPathAt(argv[0], context.RecipeDir)
```

Comment:
in PreMachine we roughly have the same code to determine the path of the script ; Probably worth havign a single utility function for it

Reply:
in a separate commit i reworked that so we do that determination only once and earlier which improves readability to me